chapters/09-package-website.Rmd

```diff
@@ -5,6 +5,77 @@
 - Getting all resources and material organized
 - Defining lifecycle of your package and individual functions (with lifecycle)
 
+## Learning Objectives
+
+* Describe the functionality `pkgdown` provides to deploy a website for package documentation
+* Customize the website with pre-existing themes
+* Organize the website with vignettes, tutorials and custom pages
+* Explain a functions' lifecycle process and the tools to handle it in R
+
+## Outline
+
+* `pkgdown`
+
+  * Text: What is `pkgdown`?
+  * Text: Benefits of having documentation be code-generated
+  * Text: Benefits of having a public website with the package docs
+  * Exercise: Build and serve the website locally
+
```

Comment:
Where do you cover the difference between serving locally and serving through GitHub Pages?  I see it in the final exercise, but it isn't clear where learners learn about it.

Reply:
I didn't cover it before. I'm adding it now in the review.